src/nvim/api/command.c

```diff
@@ -393,6 +394,13 @@ String nvim_cmd(uint64_t channel_id, Dict(cmd) *cmd, Dict(cmd_opts) *opts, Error
   VALIDATE(!is_cmd_ni(ea.cmdidx), "Command not implemented: %s", cmdname, {
     goto end;
   });
+  const char *fullname = IS_USER_CMDIDX(ea.cmdidx)
+    ? get_user_command_name(ea.useridx, ea.cmdidx)
+    : get_command_name(NULL, ea.cmdidx);
+  VALIDATE(strstr(fullname, cmdname), "Invalid command: \"%s\"", cmdname, {
```

Comment:
I think one of these makes more sense: 
```suggestion
  VALIDATE(strncmp(fullname, cmdname, strlen(cmdname)), "Invalid command: \"%s\"", cmdname, {
```
or
```suggestion
  VALIDATE(strstr(fullname, cmdname) == fullname, "Invalid command: \"%s\"", cmdname, {
```